### ISSUE TITLE
0.94.1 goldmark runtime error

### ISSUE THREAD

**Issue by killermoehre** [klakegg/docker-hugo#60]
With `0.94.1` I get the error
```text
[BUG] goldmark: runtime error: index out of range [14] with length 14: create an issue on GitHub attaching the file in: /tmp/hugo_bugs/goldmark_a451d8d2aa873e09278866fd9590d6c6.txt
```
With the tagged version `0.94.0-ext-apline-ci` it works.
With `0.94.2` local on my macOS it works as well.

**Comment by davidsneighbour** [comment]
This sounds like an issue with Goldmark itself, that was fixed with the 0.94.2 release.

**Comment by klakegg** [comment]
Is this issue related to the fact that Hugo is running inside a containerized environment, or is this simply because of [gohugoit/hugo#9658](https://github.com/gohugoio/hugo/issues/9658)?
